app/src/main/java/com/moyerun/moyeorun_android/views/BadgeRoundImageView.kt

```diff
@@ -0,0 +1,89 @@
+package com.moyerun.moyeorun_android.views
+import android.content.Context
+import android.content.res.TypedArray
+import android.util.AttributeSet
+import android.view.LayoutInflater
+import androidx.constraintlayout.widget.ConstraintLayout
+import com.moyerun.moyeorun_android.R
+import com.moyerun.moyeorun_android.databinding.ViewBadgeRoundImageBinding
+
+class BadgeRoundImageView : ConstraintLayout {
+    constructor(context: Context): super(context) { initView() }
+    constructor(context: Context, attrs: AttributeSet): super(context, attrs) {
+        initView()
+        setAttrs(attrs)
+    }
+    constructor(context: Context, attrs: AttributeSet, defStyle: Int): super(context, attrs) {
+        initView()
+        setAttrs(attrs, defStyle)
+    }
+
+    private lateinit var binding: ViewBadgeRoundImageBinding
+
+    private fun initView() { binding = ViewBadgeRoundImageBinding.inflate(LayoutInflater.from(context), this, true) }
```

Comment:
`initView()`를 생성자에서 호출하고 있던데, 이 둘을 합쳐도 될듯합니다.
```suggestion
    private val binding = 
        ViewBadgeRoundImageBinding.inflate(LayoutInflater.from(context), this, true)
```

Reply:
앗 감사합니다~~!